developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -114,21 +118,25 @@ import GraphQLFiltersOffset from '/_includes/code/graphql.filters.offset.mdx';
-- The pagination setup is not stateful. If the database state has changed between retrieving two pages there is no guarantee that your pages cover all results. If no writes happened, then pagination can be used to retrieve all possible within the maximum limit. This means asking for a single page of 10,000 objects will lead to the same results overall as asking for 100 pages of 100 results.
+- **Response time and system load increase as the number of pages grows**. Every additional page request requires a new, larger call against your collection.
+- **The cost to retrieve a page increases with each call**. For example, if your `offset` and `limit` specify results from 21-30, Weaviate retrieves 30 objects and drops the first 20. On the next call, Weaviate retrieves 40 objects and drops the first 30.
+- **Resource requirements are amplified in multi-shard configurations.** Each shard retrieves a full list of objects. Each shard also drops the objects before the offset. If you have 10 shards configured and ask for results 91-100, Weaviate retrieves 1000 objects (100 per shard) and drops 990 of them.
+- **The number of objects you can retrieve is limited**. Since pagination is resources intensive, there is a limit to how many objects you can retrieve. To change the limit, edit the `QUERY_MAXIMUM_RESULTS` environment variable. If you increase `QUERY_MAXIMUM_RESULTS`, use the lowest value possible to avoid performance problems. If the sum of `offset` and `limit` exceeds `QUERY_MAXIMUM_RESULTS`, Weaviate returns an error.
+ - **Pagination is not stateful**. If the database state changes between calls, your pages might miss results. An insertion or a deletion will change the object count. An update could change object order. However, if there are no writes the overall results set is the same if you retrieve a large single page or many smaller ones.
 
 
 ## Autocut
 
 :::info Added in `v1.20`
 :::
 
-The autocut filter limits results based on discontinuities in the result set. The filter looks for discontinuities, or jumps, in the resulting metric such as vector distances or search scores. To use autocut, specify how many jumps there should be in your query. The query stops returning results after the specified number of jumps.
+The autocut filter limits results based on discontinuities in the result set. The filter looks for discontinuities, or jumps, in the resulting metric such as vector distances or search scores.
```

Comment:
These names have been contentious. Let's call these 'operators' since it's on the additional operators page.